packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1252,7 +1252,7 @@ class RawText extends LeafRenderObjectWidget {
 
 /// A convience widget for paragraphs of text with heterogeneous style.
 ///
-/// The elements paramter is a recusrive list of lists that matches the
+/// The elements paramter is a recursive list of lists that matches the
```

Comment:
I'm amused that you fixed the typo in "recursive" but not the one in "parameter". :-)